frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -208,7 +211,11 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
         ? [dataConnection.existing]
         : [];
 
-    const pvcDetails = await createPvcDataForNotebook(projectName, storageData).catch(handleError);
+    const pvcDetails = await createPvcDataForNotebook(
+      projectName,
+      storageData,
+      storageClass?.metadata.name,
```

Comment:
if we end up renaming the const, we would then have to use i.e.

` const createdPvc = await createPvc(createData, namespace, storageClassPreferred?.metadata.name, {`

my main point being, I named the const a bit misleading back then, cause it is not the storage class name, but the instance of StorageClassKind with all related object info in it, one of the being the name ...